src/core/pubsubhub.js

```diff
@@ -75,11 +75,12 @@ export function unsub({ topic, cb }) {
 }
 
 sub("error", err => {
-  console.error(err, err.stack);
+  console.error(err, Object.fromEntries(Object.entries(err)));
```

Comment:
This is similar to doing a `console.error` followed by a `console.dir`, but within the same `console.error`.